tests/webtests/CleanOldProductionEntitiesCommandTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Copyright 2018 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Webtests;
+
+use GuzzleHttp\Psr7\Response;
+use Ramsey\Uuid\Uuid;
+use Surfnet\ServiceProviderDashboard\Domain\Entity\Entity;
+use Surfnet\ServiceProviderDashboard\Domain\Entity\Service;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class CleanOldProductionEntitiesCommandTest extends WebTestCase
+{
+
+    const NOF_PUBLISHED_ON_PRODUCTION_ENTITIES = 20;
+    const NOF_DRAFT_ON_PRODUCTION_ENTITIES = 2;
+
+    public function setUp($loadFixtures = true)
+    {
+        parent::setUp();
+
+        $this->loadFixtures();
+
+        $this->addTestEntities(self::NOF_PUBLISHED_ON_PRODUCTION_ENTITIES, self::NOF_DRAFT_ON_PRODUCTION_ENTITIES);
```

Comment:
Cool 'seeder'! Might borrow that in my tests some day. This method might be useful on the WebTestCase parent?

Reply:
I don't like the generic one-size fits all solutions for testing because there is a high change it will cost to much maintenance when something needs to get refactored later on. Maybe if we need a another seeder some day on we could split this up in separate seeders.

Reply:
This is not the first time such a solution was provided. One of the other web tests has similar functionality. That's why I gave the suggestion. For now, I'll try to remember and abstract it the next time I need some test entities!